rclcpp/include/rclcpp/clock.hpp

```diff
@@ -121,6 +123,18 @@ class Clock
     JumpHandler::post_callback_t post_callback,
     const rcl_jump_threshold_t & threshold);
 
+  /**
+   * Get current time from the time source specified by Clock::get_clock_type() const noexcept.
+   *
+   * \return RCUTILS_RET_OK if time was successfully retrieved
+   * \param now. A time point value pointer that will contain the
+   * current time from the underlying clock
+   * \throws anything rclcpp::exceptions::throw_from_rcl_error can throw.
+   */
+  RCLCPP_PUBLIC
+  rcutils_ret_t
+  get_now_as_timepoint(rcutils_time_point_value_t * now);
```

Comment:
`time_point` is two words and therefore should have an `_` between the two words, e.g. https://en.cppreference.com/w/cpp/chrono/time_point.